package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "lab0708",
+  "version": "1.0.0",
+  "description": "lab 07 08",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha, chai",
```

Comment:
This should be:

"test": "mocha" not "mocha, chai"

What would happen with the test command you've specified here is the terminal would attempt to run "mocha, chai" which isn't a proper command, so the test wouldn't work.  The terminal running "mocha" on the other hand would function appropriately.